### PR TITLE
fix: Refractored the README.md and added animated thank you gif

### PR DESCRIPTION
**Pull request by MahendraDani**
## Related Issue

Closes: #28 

## Description of Changes

Clearly and concisely describe the modifications made to successfully resolve the assigned issue. Include any pertinent information about new files or any other relevant details.
### Major changes
1. Added animated `thank-you gif` in README.md file. 

### Minor changes
1. Refractor the instructions for setting up the project locally. New contributors would be able to follow and understand the instructions easily.
2. Enhnaced the `documentation` section of the file

## Checklist:

- [x] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [x] I have included comments in areas that may be difficult to understand.
- [x] I have made corresponding updates to the project documentation.
- [x] My changes have not introduced any new warnings.

### Before

issue_carbonops_setup
### After
fix_carbonops_setup

### Before
issue_carbonops_readme_doc

### After
fix_carbonops_readme_doc

### **4.Animated thank you gif**

https://github.com/SrijanShovit/carbonops-v2/assets/116940083/83bb1232-8da3-49de-8230-80a947fb8658

### Preview gif

thankyou

### CODE REVIEW

**Comment by MahendraDani** [comment]
@shreya024 If there are any changes please let me know.
P.S : I created the gif myself
Thanks!

**Comment by SrijanShovit** [comment]
Instead of the gif, please implement an animation of at least simple left to right running thank you

**Comment by MahendraDani** [comment]
> Instead of the gif, please implement an animation of at least simple left to right running thank you

@SrijanShovit I will try!

**Comment by MahendraDani** [comment]
@SrijanShovit @shreya024 
I tried to add left to right animation as you said but it does not work in markdown files.

Please take a look at the following video-

https://github.com/SrijanShovit/carbonops-v2/assets/116940083/4667d81b-6c36-4984-8934-169dfbcac6e6

- I created a repo to test the animation.
- You can see in the video that I have used `style tags` and added animation using `@keyframes` which seems to be working in my editor.
- But this animation does not work on the remote repo. 
- Take a look at the following screenshot of my testing repo
Untitled design (2)

- You can observe that on the repo's README file no such animation is working. And the styles are always displayed which should not happen.

**Comment by MahendraDani** [comment]
@shreya024 @SrijanShovit 
I think that an animation like that can't be added in Markdown files.

**Comment by MahendraDani** [comment]
> @shreya024 @SrijanShovit 
> I think that an animation like that can't be added in Markdown files.
> 

@SrijanShovit @shreya024 I think the gif will also be fine because the animation can't be made!
Please let me know your views!
Thanks.

**Comment by MahendraDani** [comment]
@shreya024 @SrijanShovit Please review this PR!

**Comment by SrijanShovit** [comment]
@MahendraDani @shreya024 For now proceed with the gif. If later some better solution comes we shall add it.

**Comment by MahendraDani** [comment]
@SrijanShovit Alright!
Thanks!

**Comment by shreya024** [comment]
@MahendraDani PR has merge conflicts.Take a pull,resolve the conflicts and then push again.

**Comment by MahendraDani** [comment]
@shreya024 Okay I will take a pull then push it again in a minute

**Comment by MahendraDani** [comment]
@shreya024 Is everything fine from my side now? I resolved the conflicts.

**Review by shreya024 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -84,34 +84,52 @@ All
 
  
 ## Setting up the project:
-
+1. Fork this repoistory using Github GUI.
+2. Clone the project in your PC
+
+<<<<<<< HEAD
```
Comment (shreya024) [inline README.md:90]: Remove this line

File: README.md [hunk 2]
```diff
@@ -84,34 +84,52 @@ All
...
 ## Setting up the project:
-
+1. Fork this repoistory using Github GUI.
+2. Clone the project in your PC
+
+<<<<<<< HEAD
+```
+git clone https://github.com/SrijanShovit/Carbonops-v2.git
+```
+=======
```
Comment (shreya024) [inline README.md:94]: Remove this line

File: README.md [hunk 3]
```diff
@@ -84,34 +84,52 @@ All
...
 We're thrilled that you'd like to contribute to this project. Your help is essential for keeping it great. Please note we have a [Code of Conduct](./CODE_OF_CONDUCT.md), please follow it in all your interactions with the project.
 
 1. `git clone https://github.com/SrijanShovit/Carbonops-v2.git`
```
Comment (shreya024) [inline README.md:97]: Remove lines 97,98

**Review by shreya024 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (MahendraDani) [reply README.md:97]: > Remove lines 97,98

@shreya024 Removing lines 97,98

**Comment by MahendraDani** [comment]
@shreya024 Please let me know if there are any changes now!

**Pull request merged**